packages/checksum/package.json

```diff
@@ -10,9 +10,6 @@
     "test-coverage": "nyc ava",
     "debug": "NODE_ENV=test node --inspect-brk node_modules/ava/profile.js --serial tests/*.js"
   },
-  "ava": {
```

Comment:
We don't need to specify the files for Ava anymore?

Reply:
In the cases where they were removed, it was because the files being specified were the defaults that ava uses anyway: https://github.com/avajs/ava/blob/v2.1.0/docs/05-command-line.md